ccd-definition/CaseField.json

```diff
@@ -139,6 +138,14 @@
     "FieldType": "Date",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "serviceDateAndTime",
```

Comment:
This should be respondentSolicitor1ServiceDateTime

Reply:
@AlisonRevittHM if you expand this file there are actually lots more instances where it's just service, i.e serviceLocation. So I am not inclined to change all of that within this PR. Could you raise a separate JIRA ticket please